client/src/pages/cases/components/ProductReviewDate.jsx

```diff
@@ -166,11 +169,15 @@ const ProductReviewDate = ({ businessKey, processInstances }) => {
                         <td className="govuk-table__cell">No tasks</td>
                         <td className="govuk-table__cell"> </td>
                         <td className="govuk-table__cell"> </td>
+                        <td className="govuk-table__cell"> </td>
                       </tr>
                     )}
                     {formResponses.map((formResponse) => (
                       <tr key={uuidv4()} className="govuk-table__row">
                         <td className="govuk-table__cell">Record action</td>
+                        <td className="govuk-table__cell">
+                          {formResponse.data.outgoingProductType.name}
```

Comment:
Given what the failing tests were flagging up before, this would be a sensible change to make:
```suggestion
                          {formResponse.data.outgoingProductType?.name}
```